### PR TITLE
Ash toggle switch

### PR DESCRIPTION
**Pull request by sghiaa**
[Switch checkboxes to On/Off toggles](https://app.asana.com/0/584764604969369/683023916934068)

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -20,7 +20,6 @@ RUN curl -sL https://deb.nodesource.com/setup_8.x | bash - && \
     npm install -g brunch
 
 RUN npm install
-RUN brunch build --production
```
Comment (losvedir) [inline Dockerfile]: Don't need this anymore, right?

File: assets/css/signs.css [hunk 2]
```diff
@@ -15,3 +15,56 @@
...
+input[type="checkbox"].ios8-switch + label:hover:after {
+  box-shadow: 0 0 5px rgba(0,0,0,0.3);
+}
+input[type="checkbox"].ios8-switch:checked + label:after {
+  margin-left: 16px;
+}
+input[type="checkbox"].ios8-switch:checked + label:before {
+  background: #386BB2;
+}
+body {
```
Comment (losvedir) [inline assets/css/signs.css]: Hmmm, what is this for? Seems like something that shouldn't be a part of this PR.

File: Dockerfile (hunk 1 above)
Comment (sghiaa) [reply Dockerfile]: oh, hadnt pushed my last commit

File: assets/css/signs.css (hunk 2 above)
Comment (sghiaa) [reply assets/css/signs.css]: good catch - that was from a different attempt to make sliders i think

**Comment by losvedir** [comment]
Can you deploy this to Dev, just to make sure it compiles and everything and looks right?

File: lib/signs_ui_web/templates/signs/index.html.eex [hunk 3]
```diff
@@ -1,22 +1,22 @@
...
+        <b>Sign</b>
+      </div>
+    </div>
+    <%= for {sign_id, sign} <- @signs do %>
+      <%= checkbox(f, sign_id, value: sign.enabled?, id: sign_id, class: "col-md-1 col-xs-2 pull-right ios8-switch") %>
+      <%= content_tag :label, sign_id,
         for: sign_id,
-        class: if !sign.enabled?, do: "text-muted" %>
+        class: if !sign.enabled?, do: "text-muted col-md-11 col-xs-10 onoffswitch", else: "col-md-11 col-xs-10 onoffswitch" %>
+      <div class="onoffswitch">
```
[the remark this answers is not in the document]
Comment (sghiaa) [reply lib/signs_ui_web/templates/signs/index.html.eex]: ah, true - good catch

**Comment by losvedir** [comment]
Looks like there's some vertical spacing issues going on. I'm seeing:

screen shot 2018-05-25 at 9 49 05 am


Per the asana mock up the labels need to be vertically centered on the sliders.

**Pull request merged**